.github/workflows/extrinsic-ordering-check-from-bin.yml

```diff
@@ -0,0 +1,82 @@
+# This workflow performs the Extrinsic Ordering Check on demand using a binary
+
+name: Extrinsic Ordering Check from Binary
+on:
+  workflow_dispatch:
+    inputs:
+      reference_url:
+        description: The WebSocket url of the reference node
+        default: wss://kusama-statemine-rpc.paritytech.net
+        required: true
+      binary_url:
+        description: A url to a Linux binary for the node containing the runtime to test
+        default: https://github.com/paritytech/cumulus/releases/download/statemine_v3/polkadot-collator
```

Comment:
This is just an example to show the pattern. In general, users will NOT use **latest**, they will want to point at a specific version so having the version show up makes it easier. Once we have v4 released, it will be obviouls how to change the default string. If I replace with **latest**, the user will not (easily) know whether to use `3` or `v3` for instance.